### PR TITLE
include .gitattributes file

### PR DESCRIPTION
**Pull request by tphoney**

### CODE REVIEW

File: moduleroot/.gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+*.rb  eol=lf
```
Comment (igalic) [inline moduleroot/.gitattributes]: .erb, too

**Comment by tphoney** [comment]
@DavidS what do you think ?

**Comment by tphoney** [comment]
@igalic made the suggested changes.

**Comment by DavidS** [comment]
lgtm.

**Comment by bmjen** [comment]
@tphoney should also include .sh files, a rare edge case, but one that still exists I think.

**Pull request merged**